src/Form/SettingsForm.php

```diff
@@ -16,13 +15,8 @@
 final class SettingsForm extends FormBase {
   use StringTranslationTrait;
 
-  public const AUTHORITY_CVR = 'authority_cvr';
-  public const CACHE_EXPIRATION = 'cache_expiration';
-  public const CERTIFICATE = 'certificate';
+  public const ORGANISATION_BASE_API_ENDPOINT = 'organisation_base_api_endpoint';
   public const TEST_MODE = 'test_mode';
```

Comment:
Does `TEST_MODE` make sense anymore? I suppose that the API endpoint must point to a test API if we need to run in test mode.

Reply:
It does not make sense anymore. Removed.